2018-09-12 08:12:53,706 ERROR 18284 nodejs.ClusterClientNoResponseError: client no response in 30955180ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 18284
hostname: bogon

2018-09-12 08:36:15,584 ERROR 18284 nodejs.ClusterClientNoResponseError: client no response in 1361743ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 18284
hostname: bogon

2018-09-13 12:19:12,248 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 987710ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-09-13 12:39:52,627 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 1220370ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-09-13 13:01:48,891 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 1218821ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-09-13 13:18:03,656 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 913735ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-09-13 13:37:18,651 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 362790ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-09-13 19:51:01,440 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 738672ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-09-13 20:02:39,367 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 577493ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-09-13 21:57:33,453 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 1765406ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-09-13 22:41:43,674 ERROR 24991 nodejs.ClusterClientNoResponseError: client no response in 1370008ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 24991
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 01:53:20,845 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 545442ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 03:19:59,495 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 4614539ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 05:08:13,461 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 6473939ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 08:44:13,684 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 12960220ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 10:41:35,763 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 7042076ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 12:25:38,504 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 2593501ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 16:13:04,007 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 10364831ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 18:01:15,503 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 6491491ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 19:52:03,574 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 6647758ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 20:51:30,350 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 3566674ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 21:23:57,704 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 1406360ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-20 22:15:33,314 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 3095608ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 00:01:16,223 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 1041777ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 00:55:04,450 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 507724ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 00:57:19,441 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 134989ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 02:24:20,363 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 3620634ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 04:12:21,435 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 6481057ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 06:00:27,677 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 6486225ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 07:48:35,403 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 6487713ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 09:36:43,390 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 6487955ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 13:12:44,331 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 12960738ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 13:25:12,914 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 748561ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 17:30:09,690 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 14696761ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 17:41:27,951 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 678251ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 18:16:45,446 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 463509ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 19:13:56,091 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 1749938ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 19:16:08,088 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 91970ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 20:13:18,833 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 1926546ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 20:56:19,117 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 2540255ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 21:33:09,412 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 2170282ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 22:09:52,327 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 2182908ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 23:05:37,242 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 1442629ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

2018-10-21 23:57:58,880 ERROR 50918 nodejs.ClusterClientNoResponseError: client no response in 481072ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xm/ShareProject/eggPractise/egg/practise01/node_modules/_cluster-client@2.1.1@cluster-client/lib/leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 50918
hostname: xingmiaodeMacBook-Pro.local

